### PR TITLE
[net9.0] Update net9.0 with new versions

### LINKED ISSUE #19254: [ci] Add ci integration for net9.0 branch
### Description

- [x] We need to be able to build the net9.0 and produce nightly builds for testing. 
- [x] Setup maestro subscriptions
- [x] Publish to dotnet9 feed and nightly feed
- [x] Setup testing for net9 for DeviceTests and UITests

### PR DESCRIPTION
**Pull request by rmarinho**
### Description of Change

Update the net9.0 branch with needed changes

- removed the Graphics.Mac and Graphics.Gtk
- Update WindowsAppSdk

fixes #19254

### CODE REVIEW

File: Microsoft.Maui.sln [hunk 1]
```diff
@@ -207,28 +207,18 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphicsTester.Android", "s
...
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphicsTester.Skia.Mac", "src\Graphics\samples\GraphicsTester.Skia.Mac\GraphicsTester.Skia.Mac.csproj", "{5E8D70F3-990D-4B40-8B13-AA818F21E8A0}"
-EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphicsTester.Skia.Tizen", "src\Graphics\samples\GraphicsTester.Skia.Tizen\GraphicsTester.Skia.Tizen.csproj", "{20756558-1284-42D6-AB7B-3E59DF8A6B34}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphicsTester.Skia.Windows", "src\Graphics\samples\GraphicsTester.Skia.Windows\GraphicsTester.Skia.Windows.csproj", "{8441FA70-9041-4727-91B9-2F39881CC0EE}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphicsTester.WinUI.Desktop", "src\Graphics\samples\GraphicsTester.WinUI.Desktop\GraphicsTester.WinUI.Desktop.csproj", "{4BF0AF30-6E18-47D8-B90C-E99C1917BEF5}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Graphics.Skia.GtkSharp", "src\Graphics\src\Graphics.Skia.GtkSharp\Graphics.Skia.GtkSharp.csproj", "{AFD9D653-08C4-456C-AA1B-F5C9F621D655}"
```
Comment (rmarinho) [inline Microsoft.Maui.sln:226]: @mattleibow  can you check if we can workaround removing these projects from the sln ?

File: src/Compatibility/Core/src/Compatibility.csproj [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <iOSRoot>iOS\</iOSRoot>
     <WindowsRoot>Windows\</WindowsRoot>
     <TizenRoot>Tizen\</TizenRoot>
-    <IsTrimmable Condition="!$(TargetFramework.StartsWith('netstandard'))">true</IsTrimmable>
+    <IsTrimmable>false</IsTrimmable>
```
Comment (jonathanpeppers) [inline src/Compatibility/Core/src/Compatibility.csproj:11]: We shouldn't do this, as it will significantly increase app sizes. What was breaking?

**Review by jonathanpeppers — COMMENTED** [review]
Everything else looks, OK, just had the one app size concern. 👍

File: Microsoft.Maui.sln (hunk 1 above)
Comment (rmarinho) [reply Microsoft.Maui.sln:226]: https://github.com/dotnet/maui/issues/19428

File: src/Compatibility/Core/src/Compatibility.csproj [hunk 3]
```diff
@@ -16,7 +16,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="$(TargetFramework.Contains('-windows')) == true ">
-    <NoWarn>$(NoWarn);CA1416;CS8305</NoWarn>
+    <NoWarn>$(NoWarn);CA1416;CS8305;IL2059</NoWarn>
```
Comment (jonathanpeppers) [inline src/Compatibility/Core/src/Compatibility.csproj:19]: Can we do `IL2059` conditionally on Windows? It is a useful warning on iOS/Android.

Not a blocker, though.

**Review by jonathanpeppers — APPROVED** [review]
This goods good to me if CI is green. 👍

**Pull request merged**